tests/ct_connection.c

```diff
@@ -0,0 +1,176 @@
+#include "adapters/libevent.h"
+#include "hircluster.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define CLUSTER_IP_WITH_PASSWORD "127.0.0.1:30001"
```

Comment:
Since it is IP and port and it's passed to a function called `...AddNodes()`, maybe the name could be CLUSTER_NODE_WITH_PASSWORD instead.

Reply:
Done